kubernetes/core/src/main/kotlin/com/google/kubernetes/tools/core/util/CoreBundle.kt

```diff
@@ -20,7 +20,7 @@ import com.intellij.CommonBundle
 import org.jetbrains.annotations.PropertyKey
 import java.util.ResourceBundle
 
-private const val BUNDLE_NAME = "messages.CoreBundle"
+private const val BUNDLE_NAME = "messages.KubectlBundle"
```

Comment:
I think we can keep `CoreBundle` name, for most of our modules, we use the scheme `[module-name]Bundle`